test/core/scale/CMakeLists.txt

```diff
@@ -79,3 +79,10 @@ addtest(scale_convenience_functions_test
 target_link_libraries(scale_convenience_functions_test
     scale
     )
+
+addtest(scale_tuple_test
```

Comment:
Please set your CMake settings as here and fix formatting https://soramitsu.atlassian.net/wiki/spaces/FIL/pages/1255309333/CLion+preparation+before+development#CMake-code-style